src/entities/Place/utils.ts

```diff
@@ -66,3 +68,13 @@ export function createPlaceFromDeployment(
     ...data,
   }
 }
+
+export function placeTargetUrl(
+  place: Pick<PlaceAttributes, "base_position">
+): string {
+  const target = new URL(DECENTRALAND_URL)
+  target.pathname = ""
```

Comment:
if we have to ensure a root URL we can do it only with one line

```suggestion
  const target = new URL('/', DECENTRALAND_URL)
```